packages/components/src/ui/ariakit-tooltip/style.scss

```diff
@@ -0,0 +1,14 @@
+.components-ariakit-tooltip {
```

Comment:
I replaced emotion with sass so the `z-index` function could be utilized and because we may be moving away from emotion.